doc/deploy/deployment.md

```diff
@@ -108,24 +108,50 @@ All Helm parameters are described in the table at the bottom of this section.
+--set imagePullSecret.password=<REPLACE_WITH_REGISTRY_TOKEN_GENERATED_IN_PREREQs_STAGE1_REGISTRY.d)>  \
+--set localConfigName=<Local Kubernetes Config File for Current Cluster>  \
+--set volumes.hostPath=<Host_Path_location_of_local_Kubernetes_config_file> \
+--set configMap.name=mcad-controller-configmap \
+--set configMap.quotaEnabled='"false"' \
+--set coscheduler.rbac.apiGroup="scheduling.sigs.k8s.io" \
+--set coscheduler.rbac.resource="podgroups"
 ```
 
 ##### Example 1
-*Assuming the default for `image.repository` and `image.tag` fields*:
+*Deploying MCAD using built images*:
+
 ```
-helm install mcad-controller --namespace kube-system
+helm upgrade --install --wait mcad . --namespace kube-system --set image.pullPolicy=IfNotPresent --set image.repository=darroyo/mcad-controller --set image.tag=quota-management-v1.29.40
+
 ```
+
 ##### Example 2
-*Assuming the MCAD controller image is already pulled onto the local target machine with the following image `image.repository=mcad-controller`, `image.tag=latest`*
+*Assuming the default for `image.repository` and `image.tag` fields*:
 ```
-helm install mcad-controller --namespace kube-system --wait --set image.pullPolicy=Never --set image.repository=mcad-controller --set image.tag=latest
+helm upgrade --install --wait mcad . --namespace kube-system
 ```
 ##### Example 3
+*Assuming the MCAD controller image is already pulled onto the local target machine with the following image `image.repository=mcad-controller`, `image.tag=latest`*
+```
+helm upgrade --install --wait mcad . --namespace kube-system --set image.pullPolicy=Never --set image.repository=mcad-controller --set image.tag=latest
+```
+##### Example 4
 To adjust the cpu and memory demands of the deployment with command line overrides example:
 
 ```
-helm install mcad-controller --namespace kube-system --wait --set resources.requests.cpu=1000m --set resources.requests.memory=1024Mi --set resources.limits.cpu=1000m --set resources.limits.memory=1024Mi --set image.repository=myDockerReegistry/mcad-controller --set image.tag=latest --set image.pullPolicy=Always
+helm upgrade --install --wait mcad . --namespace kube-system --wait --set resources.requests.cpu=1000m --set resources.requests.memory=1024Mi --set resources.limits.cpu=1000m --set resources.limits.memory=1024Mi --set image.repository=myDockerReegistry/mcad-controller --set image.tag=latest --set image.pullPolicy=Always
+```
+
+##### Example 5
+MCAD supports bring-you-own scheduler use case. To install MCAD with co-scheduler use the follwing command:
+
+```
+helm upgrade --install --wait mcad . --namespace kube-system  --set loglevel=4  --set image.repository=$IMAGE  --set image.tag=quota-management-v1.29.40  --set image.pullPolicy=IfNotPresent  --set configMap.name=mcad-controller-configmap  --set configMap.quotaEnabled='"false"'  --set coscheduler.rbac.apiGroup="scheduling.sigs.k8s.io"  --set coscheduler.rbac.resource="podgroups"
```

Comment:
what is the value of `$IMAGE`?

Reply:
Thanks, Added export command : `export IMAGE=darroyo/mcad-controller`